addons/website/static/tests/tours/page_manager.js

```diff
@@ -29,6 +29,27 @@ const checkKanbanGroupBy = [{
     trigger: '.o_cp_searchview .o_facet_remove',
 }];
 
+const checkWebsiteFilter = [
+    {
+        content: "Click on My Website",
+        trigger: "button.dropdown-toggle:contains('My Website')",
+    }, {
+        content: "Select My Website 2",
+        trigger: ".dropdown-menu.show > .dropdown-item:contains('My Website 2')",
+    }, {
+        content: "Check that the homepage is now the one of My Website 2",
+        trigger: ".o_list_table .o_data_row .o_data_cell[name=name]:contains('Home') " +
+                "~ .o_data_cell[name=website_id]:contains('My Website 2')",
+        run: () => null, // it's a check
+    }, {
+        content: "Click on My Website 2",
+        trigger: "button.dropdown-toggle:contains('My Website 2')",
+    }, {
+        content: "Go back to My Website",
+        trigger: ".dropdown-menu.show > .dropdown-item:contains('My Website')",
+    },
```

Comment:
With this step, which is there to restore the initial state for the rest of the tour, isn't there a room for a race condition that would click on the Homepage (next step) before the homepage is actually replace from the website 2 one to the website 1 one?
I'd have expected the same condition as above to check that the homepage is now correctly set back to the website 1 one.